content/influxdb/cloud/reference/cli/influx/ping/_index.md

```diff
@@ -1,17 +0,0 @@
----
-title: influx ping
-description: >
-  The `influx ping` command checks the health of a running InfluxDB instance by
-  querying the `/health` endpoint.
-menu:
-  influxdb_cloud_ref:
-    name: influx ping
-    parent: influx
-weight: 101
-influxdb/cloud/tags: [ping, health]
-related:
-  - /influxdb/cloud/reference/cli/influx/#provide-required-authentication-credentials, influx CLI—Provide required authentication credentials
-  - /influxdb/cloud/reference/cli/influx/#flag-patterns-and-conventions, influx CLI—Flag patterns and conventions
----
-
-{{< duplicate-oss >}}
```

Comment:
We shouldn't delete this doc. The `ping` command is still part of the `influx` CLI, so users will still see it in the help output. Instead, just put a note or cloud message at the top above `{{< duplicate-oss >}}`. You can see an example here: https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/backup/

Here's the source for that page ^ https://raw.githubusercontent.com/influxdata/docs-v2/master/content/influxdb/cloud/reference/cli/influx/backup/_index.md